apps/trackers/product_definition_handlers/ubi_handler.py

```diff
@@ -16,6 +16,7 @@ def __init__(self) -> None:
 
     def has_ubi_packages(ps_module: PsModule, affect: Affect) -> bool:
         """check weheter a ps_module has ubi packages given a target PsUpdateStream name"""
+        # TODO: I can't find any code that sets special_handling_features. Is this missing and should we file a Jira?
         if "ubi_packages" not in ps_module.special_handling_features:
```

Comment:
@osoukup I can't find any code that sets `special_handling_features`. Is this missing and should we file a Jira?

Reply:
Looks like a bug :lady_beetle:

Reply:
Filed as OSIDB-2161